### ISSUE TITLE
Incorrect reporting on role=combobox & aria-expanded missing?

### ISSUE THREAD

**Issue by Volker-E** [dequelabs/axe-core#406]
axe-core v2.3.1 with axe-devtools v3.0.0 reports critical “Required ARIA attributes must be provided” on following markup:
```lang=html
<span tabindex="0" aria-disabled="false" class="oo-ui-dropdownWidget-handle" role="combobox" aria-owns="oojsui-45" aria-autocomplete="list">
```

Both, [ARIA 1.0](https://www.w3.org/TR/wai-aria/roles#combobox) and [1.1](https://www.w3.org/TR/wai-aria-1.1/#combobox) are mentioning an implicit default value `false` for `aria-expanded`. ARIA 1.1 is a bit less ambiguous and states “Authors **must** set `aria-expanded` to `true` on an element with role `combobox` when it is expanded and `false` when it is collapsed.”
In this reading one could argue that the attribute needs to be set explicitely not earlier than when the expanded state is changed. 

Therefore the error reporting by aXe seems incorrect.

**Comment by WilcoFiers** [comment]
Agreed. I suspect this might not have been put in initially because of limited support, but some quick testing with VO and NVDA shows it to work.
